src/app/features/issue/providers/github/github.const.ts

```diff
@@ -97,6 +98,14 @@ export const GITHUB_CONFIG_FORM: LimitedFormlyFieldConfig<GithubCfg>[] = [
       label: T.F.GITHUB.FORM.IS_ASSIGNEE_FILTER,
     },
   },
+  {
+    key: 'importLabelsAsTags',
```

Comment:
For booleans the current code base almost always uses an `is` prefix e.g. `isSomethingTrue`.

Reply:
addressed in 67e18a659de96def3d58c68a59edc7ba981b789b